framework/test/jest/unit/specs/modules/chain/dpos/getRoundDelegates/utils.js

```diff
@@ -0,0 +1,24 @@
+const { hash } = require('@liskhq/lisk-cryptography');
+
+const shuffleDelegateListForRound = (round, list) => {
```

Comment:
Shouldn't we avoid having implementation to test implementation? Maybe here we should use a known list instead of re-implementing the `shuffleDelegateListForRound`

Reply:
delegateList changes by roundNumber and to sake of avoiding having hardcoded values in tests I'm generating round number randomly. Thus, I wanted to have the shuffle logic as utils. Also I wanted to avoid using original implementation just in case something breaks there. 

I think this is a better way to go and maybe mowing shuffle logic to elements is the way to go?

Reply:
This shuffle logic is essential part of the DPOS module. I don't think it make sense to move to any other parts.
I also think instead of generating the implementation by implementation, it's better to use fixture in this case
Since it's very hard to test, we should add specific test for this function (although it's not public function) with fixture.